content/ecosystem/data.toml

```
@@ -502,7 +502,8 @@ categories = ["audio"]
 [[items]]
 name = "miniquad"
 source = "crates"
-categories = ["2drendering"]
+categories = ["3drendering"]
+homepage = "https://github.com/not-fl3/miniquad"
```

This needs to be `homepage_url` for the site to pick it up:
```suggestion
homepage_url = "https://github.com/not-fl3/miniquad"
```

---

Fixed, thanks!